cmd/run/flags.go

```diff
@@ -137,12 +137,18 @@ func bindRunFlagsFunc(flags *pflag.FlagSet) func(*cobra.Command, []string) {
 		util.MustBindPFlag("maxTypesPerAuthorizationModel", flags.Lookup("max-types-per-authorization-model"))
 		util.MustBindEnv("maxTypesPerAuthorizationModel", "OPENFGA_MAX_TYPES_PER_AUTHORIZATION_MODEL", "OPENFGA_MAXTYPESPERAUTHORIZATIONMODEL")
 
+		util.MustBindPFlag("maxConcurrentReads", flags.Lookup("max-concurrent-reads"))
```

Comment:
This belongs under the family of the datastore flags. Please add the datastore prefix.

`--datastore-max-concurrent-reads`.

Reply:
Hmmm. The `datastore` flag controls config of the SQL driver, but this flag is used in application logic. Unless we change the code so that commands have access to database configuration (which they don't have access to right now), I still need this flag to be part of this struct 

```
type server.Config struct {
	ResolveNodeLimit        uint32
	ResolveNodeBreadthLimit uint32
	MaxConcurrentReads      uint32
	ChangelogHorizonOffset  int
	ListObjectsDeadline     time.Duration
	ListObjectsMaxResults   uint32
	Experimentals           []ExperimentalFeatureFlag
}
```

I could add an interface method here:

```

type OpenFGADatastore interface {
	TupleBackend
	AuthorizationModelBackend
	StoresBackend
	AssertionsBackend
	ChangelogBackend

	IsReady(ctx context.Context) (bool, error)

	Close()

        MaxConcurrentReadsAllowed()
}
```

 WDYT?

Reply:
It controls `storage` layer parameters though (e.g. the `limiter` in `pkg/storage/storagewrappers`).

If we're going to leave as is, I vote we at least make it specific to the endpoint.

`--check-max-concurrent-reads`
`--list-objects-max-concurrent-reads`
`--expand-max-concurrent-read`

because those name make it more clear that it's controlling graph resolution behavior.